### PR TITLE
[general] new bundle for Debian 9, 10, 11 and AstraLinux 2.12 (Orel)

### PR DESCRIPTION
**Pull request by RomanenkoDenys**
## Description

New bundle supporting Debian 9, 10, 11 and AstraLinux 2.12(Orel).
Based on https://github.com/deckhouse/deckhouse/pull/397.
## Why do we need it, and what problem does it solve?

We need support of wide range of linux distros.
## Changelog entries

```changes
module: general
type: feature
description: "New bundle supporting Debian 9, 10, 11 and AstraLinux 2.12(Orel)"
note:
```

### CODE REVIEW

File: modules/040-node-manager/openapi/config-values.yaml [hunk 1]
```diff
@@ -29,11 +29,12 @@ properties:
     default:
       - "ubuntu-lts"
       - "centos-7"
+      - "debian"
     x-examples:
       - ["ubuntu-lts"]
     items:
       type: string
-      enum: [ubuntu-lts, centos-7]
+      enum: [ubuntu-lts, centos-7, debian]
```
Comment (apolovov) [inline modules/040-node-manager/openapi/config-values.yaml:37]: astra?

File: modules/040-node-manager/template_tests/module_test.go [hunk 2]
```diff
@@ -72,6 +72,7 @@ const nodeManagerConfigValues = `
 allowedBundles:
   - "ubuntu-lts"
   - "centos-7"
+  - "debian"
```
Comment (apolovov) [inline modules/040-node-manager/template_tests/module_test.go:75]: astra?

**Review by apolovov — DISMISSED** [review]

File: modules/040-node-manager/openapi/config-values.yaml (hunk 1 above)
Comment (RomanenkoDenys) [reply modules/040-node-manager/openapi/config-values.yaml:37]: Astra discovered as debian, so bundle isn't needed

File: modules/040-node-manager/template_tests/module_test.go (hunk 2 above)
Comment (RomanenkoDenys) [reply modules/040-node-manager/template_tests/module_test.go:75]: Astra discovered as debian, so bundle isn't needed

**Comment by nabokihms** [comment]
@konstantin-axenov promised to take a look soon.

**Comment by konstantin-axenov** [comment]
Everything is fine. After rebase and squash, it can be merged.

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**